packages/kit/test/apps/basics/test/client.test.js

```diff
@@ -698,28 +698,36 @@ test.describe('Prefetching', () => {
 		page
 	}) => {
 		await page.goto('/routing/a');
-		await app.prefetch('/routing/prefetched/hash-route#please-dont-show-me');
-		await app.goto('/routing/prefetched/hash-route');
+		await app.prefetch('/routing/prefetching/hash-route#please-dont-show-me');
```

Comment:
i might just not be fully caffeinated yet but the diff shows a bunch of files that were deleted but are still referenced in these tests? somewhat confused as to how they're passing!

Reply:
oh yeah sorry, forgot to mention that. While working on the test I noticed that there was essentially some duplicate pages for the tests. I removed them and changed the urls of the affected tests (the contents of those deleted files were the same as the ones they point to now).

Reply:
oooohhhhhh huh. probably my bad then — oops!